packages/forms-web-app/src/routes/appeal-householder-decision/index.js

```diff
@@ -4,14 +4,17 @@ const router = express.Router();
 const checkDecisionDateDeadline = require('#middleware/check-decision-date-deadline');
 const checkAppealExists = require('#middleware/check-appeal-exists');
 const checkLoggedIn = require('#middleware/check-logged-in');
+const fetchExistingAppealMiddleware = require('#middleware/fetch-existing-appeal');
+const {
+	getAppealAlreadySubmitted
+} = require('../../controllers/full-appeal/submit-appeal/appeal-already-submitted');
 
-// pre login
-router.use(checkAppealExists, checkDecisionDateDeadline, require('./planning-application-number'));
 router.use(require('./cannot-appeal'));
-
-// login
 router.use(require('./login'));
 
+router.get('/appeal-already-submitted', [fetchExistingAppealMiddleware], getAppealAlreadySubmitted);
```

Comment:
Is this a new route?

Reply:
yes I hit a 404 on this path for householder while testing, so just added the same page from full planning
Could take out of this PR and make a separate ticket

Reply:
I'm OK with it being in here.